src/B_read_from_files.py

```diff
@@ -611,6 +621,7 @@ def from_file(project_site, path_from):
                     project_site.update(
                         {dictionary_title: pd.Series([0 for i in range(0, 8760)])}
                     )
+            logging.debug(f"read timeseries four{dictionary_title}")
```

Comment:
You should make a nice debug message out of this. 
```suggestion
            logging.debug(f"Data input: {column_item} will be read from column {dictionary_title}")
```